tests/test_create_conference.py

```diff
@@ -0,0 +1,339 @@
+        invitation = openreview.Invitation(
+            id = 'DEF.cc/-/Submission',
+            writers = ['openreview.net'],
+            signatures = ['openreview.net'],
+            readers = ['everyone'],
+            invitees = ['everyone'],
+            reply=reply
+        )
+        client.post_invitation(invitation)
+        assert client.get_invitation('DEF.cc/-/Submission')
+
+        # create_dataset submission invitation to test deduplication
+        invitation = openreview.Invitation(
+                id = 'DEF.cc/-/Blind_Submission',
+                writers = ['openreview.net'],
+                signatures = ['openreview.net'],
+                readers = ['everyone'],
+                invitees = ['everyone'],
+                reply=reply
+            )
+        client.post_invitation(invitation)
+        assert client.get_invitation('DEF.cc/-/Blind_Submission')
+
+        # create_dataset edge invitation to test exclusion
+        invitation = openreview.Invitation(
+                id = 'DEF.cc/-/Expertise_Selection',
+                writers = ['openreview.net'],
+                signatures = ['openreview.net'],
+                readers = ['everyone'],
+                invitees = ['everyone'],
+                reply={
+                    "forum": None,
+                    "replyto": None,
+                        "writers": {
+                            "values": [
+                                "openreview.net"
+                            ]
+                        },
+                        "signatures": {
+                            "description": "How your identity will be displayed with the above content.",
+                            "values": [
+                                "openreview.net"
+                            ]
+                        },
+                        "readers": {
+                            "description": "The users who will be allowed to read the above content.",
+                            "values": [
+                                "everyone"
+                            ]
+                        },
+                        "content": {
+                            "head": {
+                                "type": "Note"
+                            },
+                            "tail": {
+                                "type": "Profile"
+                            },
+                            "label": {
+                                "value-radio": [
+                                    "Exclude"
+                                ],
+                                "required": False
+                            }
+                        }
+                }
+            )
+        client.post_invitation(invitation)
+        assert client.get_invitation('DEF.cc/-/Expertise_Selection')
+
+    def test_post_submissions(self, client, openreview_client):
+        
+        def post_notes(data, data_invitation, api_invitation):
```

Comment:
what's the difference between `data_invitation` and `api_invitation`?

Reply:
The fake notes are stored in the JSON as:
```
{
"notes": {
     "conf_id/-/submission": [ ... ]
}
```
so the distinction is that the fake data stored on disk is formatted differently from the invitation that they will be posted to. The real solution here that I'll work on would be to just store the data exactly as it would be posted